sharktank/sharktank/utils/iree.py

```diff
@@ -22,7 +23,28 @@
 from .tree import Tree
 
 
-def get_iree_devices(driver: str, device_count: int) -> List[iree.runtime.HalDevice]:
+def get_iree_devices(
```

Comment:
May also want to put pytest details in https://github.com/nod-ai/shark-ai/blob/main/docs/developer_guide.md#running-tests

Reply:
I would, but unfortunately our code is fragmented and not all tests use this function to create the devices.
I would need to refactor them first to use this function.